WordPress/Classes/ViewRelated/Stats/Insights/SimpleTotalsCell.swift

```diff
@@ -1,32 +1,31 @@
 import UIKit
 
-class SimpleTotalsCell: UITableViewCell {
+class SimpleTotalsCell: UITableViewCell, NibLoadable {
```

Comment:
I love seeing `NibLoadable` spread!

Reply:
I'm loving it too! It's incredibly handy.